utils.js

```diff
@@ -0,0 +1,9 @@
+export function compareNumbers(guess, correctNumber) {
+
```

Comment:
this could be considered a styling thing but this is called whitespace. If this is your style, just ignore this comment. Otherwise, you don't need the extra whitespace, just have your if statements one after the other.